src/main/java/edu/carrollcc/cis232/Question1.java

```diff
@@ -1,10 +1,35 @@
 package edu.carrollcc.cis232;
 
+//Write a static method named maxElement in edu.carrollcc.cis232. Question1 class
+//that takes only an int[] as an argument and returns the largest value. The method
+//should use recursion to find the largest element. If the int[] argument has no
+//elements or is null, throw an IllegalArgumentException. Demonstrate the method in
+//the Question1 main method.
+//HINT: Pass the max element as an argument in the recursive call. If the next value
+//beats it, replace the max for the next recursive call.
+
 public class Question1 {
 
 	public static void main(String[] args) {
-
+		int[] numbers = {1,2,3,4,5,6,7,8,9,0};		//call
+		System.out.println(maxElement(numbers));
 	}
 
-	//Write static maxElement method here
-}
+	public static int maxElement(int[] numbers){
+		if(numbers == null || numbers.length == 0){		//b
+			throw new IllegalArgumentException();
+		}
+		return maxElement(numbers, numbers.length - 1, 0, 0);
```

Comment:
your value for startIndex should be 0 and your value for endIndex should be numbers.length - 1.  The value for `m` should be the first value of the array.  (-9 Technical failed 3 unit tests)